### DISCUSSION TITLE
Feature Proposal: Automatic Restart of Queue Worker on Job Edits

### DISCUSSION THREAD

**Discussion by dasundev** [laravel/framework#53069] [General] [answered]
We know that when editing a queue job, we must restart the worker during development, which can be quite disruptive as it requires extra effort. Don’t you think it should restart automatically? I would like to contribute this feature if the community would appreciate it.

cc @taylorotwell @nunomaduro @driesvints

**Comment by henzeb** [comment, marked as answer]
We already have that. `queue:listen`. https://laravel.com/docs/11.x/queues#the-queue-work-command

**Reply by dasundev** [reply]
I missed that. 🤦‍♂️
